radio/sdcard/horus/WIDGETS/BattCheck/main.lua

```diff
@@ -446,6 +485,7 @@ local function refresh(wgt)
   elseif wgt.zone.w  > 150 and wgt.zone.h >  28 then refreshZoneSmall(wgt)
   elseif wgt.zone.w  >  65 and wgt.zone.h >  35 then refreshZoneTiny(wgt)
   end
+  lcd.drawText(wgt.zone.x, wgt.zone.y, string.format("r:%d", wgt.telemResetCount), SMLSIZE + CUSTOM_COLOR + RIGHT) -- ?????????????????????????
```

Comment:
why the ???? ?

Reply:
Mistake, thanks!!
will be removed.
that was a number to help debugging that the mechanism of detecting reset event is working